### ISSUE TITLE
Usage instructions should mention <-- toc --> marker at the beginning

### ISSUE THREAD

**Issue by dbooth-boston** [jonschlinkert/markdown-toc#99]
It would help if the usage instructions said up front said something like: 

> 1. Add the following line to your markdown file where you want your TOC inserted:
> ```<-- toc -->```
> 2. Run markdown-toc:
> ...

This information does appear much later in the README.md, but it would help to have it right up front also.

Let me know if I should submit a Pull Request for this.

**Comment by doowb** [comment]
It's listed in the [CLI usage](https://github.com/jonschlinkert/markdown-toc#cli) section under the `-i` flag since it's only used when "inserting" the TOC into the markdown file.

Feel free to submit a PR to clarify this since other people may miss that point too. There could be a "Quick start" section before "CLI" or something like that, that just shows a simple markdown file and the command to run.

---

## Quick start

Create a markdown file: **README.md**

```markdown
<!-- toc -->
# A

## aa

something in aa

## ab

something in ab
```

Run the `markdown-toc` command:

```sh
markdown-toc -i README.md
```
---

Something like that might provide some more information.

**Comment by doowb** [comment]
This was resolved in #100
